Microsoft.Recognizers.Text.Number.Tests/TestDecimalAndThousandsSeparators.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Globalization;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Immutable;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Number;
+
+namespace Microsoft.Recognizers.Text.Number.Tests
+{
+    public class ArabicNumberTestConfiguration : INumberParserConfiguration
+    {
```

Comment:
It seems that we do not need to create a new configuration.We can use https://github.com/Microsoft/Recognizers-Text/blob/master/Microsoft.Recognizers.Text.Number/English/Parsers/EnglishNumberParserConfiguration.cs